essentials-framework/Essentials DM/Essentials_DM/Chassis/DmBladeChassisController.cs

```diff
@@ -417,23 +477,44 @@ void AddInputPortWithDebug(uint cardNum, string portName, eRoutingSignalType sig
         }
 
 
-        /// <summary>
-        /// Adds OutputPort
-        /// </summary>
-        void AddOutputPortWithDebug(string cardName, string portName, eRoutingSignalType sigType, eRoutingPortConnectionType portType, object selector) {
+        
+        /*void AddOutputPortWithDebug(string cardName, string portName, eRoutingSignalType sigType, eRoutingPortConnectionType portType, object selector) {
             var portKey = string.Format("{0}--{1}", cardName, portName);
             Debug.Console(2, this, "Adding output port '{0}'", portKey);
             OutputPorts.Add(new RoutingOutputPort(portKey, sigType, portType, selector, this)
             {
                 FeedbackMatchObject = Chassis.Outputs[(uint)selector]
             });
+        }*/
+
+        /// <summary>
+        /// Adds OutputPort
+        /// </summary>
+        void AddOutputPortWithDebug(uint cardNum, string portName, eRoutingSignalType sigType, eRoutingPortConnectionType portType, object selector)

```

Comment:
Modified this method and the method that calls it to bring it in line with AddInputPortWithDebug.

There was an an invalid cast exception with the original method, which is odd, because to my knowledge, this class hadn't been changed since I deployed it back in November 2019.

Reply:
Thanks @TrevorPayne. Appreciated